test/helpers/kubectl.go

```diff
@@ -800,46 +800,50 @@ func (kub *Kubectl) CiliumEndpointsListByLabel(pod, label string) (EndpointMap,
 	return result, nil
 }
 
-// CiliumEndpointWait waits until all endpoints managed by the specified Cilium
-// pod are ready. Returns false if the command to retrieve the state of the
-// endpoints times out.
-func (kub *Kubectl) CiliumEndpointWait(pod string) bool {
+// CiliumEndpointWaitReady waits until all endpoints managed by all Cilium pods
+// pod are ready. Returns an error if cannot get the cilium pods, or the
```

Comment:
"pods pod are ready" --> "pod are ready"
"Returns an error if the Cilium pods cannot be retrieved via Kubernetes, or endpoints are not ready after a specified timeout"